src/Nox.Types/Types/Image/ImageFormatType.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+
+namespace Nox.Types;
+
+/// <summary>
+/// Represents the format type of an image.
+/// </summary>
+public enum ImageFormatType
```

Comment:
Can we create a custom enumeration class, that would have Id, Name and Extension properties so that all logic within this file (multiple classes) are within a single class? Besides the different Types, we could also expose a SupportedExtensions property (or similar method) that would have the same logic as the one in ImageTypeOptions - having everything related to image types in one class.

You can take the MeasurementUnit as an example: https://github.com/NoxOrg/Nox.Generator/blob/main/src/Nox.Types/Types/Volume/MeasurementUnit.cs

Here is some additional reference: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum

We can create a base Enumeration.cs class and implement it by MeasurementUnit and ImageFormatType (now or in some future refactoring).

Reply:
Required changes was applied